src/renderer/components/dex/OrderBook.vue

```diff
@@ -1,5 +1,15 @@
 <template>
   <section id="dex--order-book">
+    <div class="color-mode-btn" @click="toggleNightMode">
```

Comment:
Putting this in this component seems reasonable for now since otherwise more routing would be required, and it was in the market selector component before.